slasher/rpc/slashing_bench_test.go

```diff
@@ -0,0 +1,137 @@
+package rpc
```

Comment:
I'd rename this file to `slashing_benchmark` Since It's just benchmark there's no test

Reply:
The file should end in `_test.go` so that go tools knows not to include it in production binary